### ISSUE TITLE
Overhead for audio only conference 

### ISSUE THREAD

**Issue by fluffy** [kixelated/warp-draft#131]
We need to work to get the number of bytes sent for a use case like clubhouse to relatively efficient. ( I am thinking something similar to RTP )

**Comment by ianswett** [comment]
I marked this NotTransport because I don't believe this is a transport issue.  Unless this is an issue implying moq-transport should support datagrams, in which case let's say so explicitly.

**Comment by kixelated** [comment]
> I marked this NotTransport because I don't believe this is a transport issue. 

+1

> Unless this is an issue implying moq-transport should support datagrams, in which case let's say so explicitly.

Datagrams would slightly increase the overhead actually, since QUIC can coalesce multiple STREAM frames into a single packet.

**Comment by afrind** [comment]
Individual Comment:

+1 that this measurement is very useful, an interested party asked me for the same.  

Can we assign someone who is able to send audio over moqt-01 (perhaps @suhasHere or @jordicenzano) to compute the current overhead with their implementation?  This data may inform further work we need to do.

**Comment by afrind** [comment]
@suhasHere can you measure this in your implementation and report back

**Comment by suhasHere** [comment]
@afrind  Yes . I might need some time to get the data reported. but happy to instrument the media flows.

**Comment by afrind** [comment]
I'm just going to close this.  We've compressed datagrams as much as we possibly can.  The minimum MOQT overhead for a datagram is Track Alias + Group ID.  With < 127 tracks, you can can send up to 16k datagrams with 3 bytes of overhead (1 byte alias, 2 byte group).
